torch/csrc/cuda/Module.cpp

```diff
@@ -1162,6 +1162,13 @@ static void registerCudaPluggableAllocator(PyObject* module) {
             ->release_storage_and_set_meta_custom_data_ptr_error_msg_(s);
       });
 
+  m.def(
+      "_set_storage_data_ptr_access_error_msg",
```

Comment:
How come this name doesn't mention that it releases data?

Reply:
It doesn't release data though. This only sets an error message